vendor/engines/projects/app/models/projects/project.rb

```diff
@@ -0,0 +1,15 @@
+module Projects
+
+  class Project < ActiveRecord::Base
+
+    belongs_to :facility, foreign_key: :facility_id
+    attr_accessible :description, :name
```

Comment:
We have strong_params gem installed. Can we use that instead? Add `include ActiveModel::ForbiddenAttributesProtection` to the model. One less model we have to deal with when we upgrade rails.